### ISSUE TITLE
Ignore this! Since 2.9.0 station departures aren't loading

### ISSUE THREAD

**Issue by Der-Flip** [jheubuch/traewelling-android#378]
Edit 3: I want to apologize. For some reason ignore updates was active since 1.20.1 on which I still was. Updated now and all good. I apologize for my non-functioning brain

~~Edit: I just noticed that there's a 2.10.2 already, which isn't yet available on f-droid...~~

~~Edit 2: so I noticed the ignore all new versions was active for some reason, but now that this is deactivated it wants me to "update" from 2.9.0 to 1.20.1... I know this isn't all träweldroid connected but i am confused as hell now :'D~~

~~**Describe the bug**
When selecting any station the departures are not loading, instead there is an infinite loading screen~~

~~**To Reproduce**
1. Select any station
2. Attempt to view departures
3. Experience mesmerizing loading animation~~

~~**Expected behavior**
As in previous versions have the departures displayed~~

~~**Smartphone (please complete the following information):**
 - Device: redmi 9c
 - OS: Android 10 qp1a.190711.020
 - app-Version 2.9.0 (android version see above)~~

~~**Additional context**
basically makes app unusable :(
~~
